codecov_auth/commands/owner/interactors/save_terms_agreement.py

```diff
@@ -22,12 +23,20 @@ class SaveTermsAgreementInteractor(BaseInteractor):
     def validate(self, input: TermsAgreementInput):
         if input.terms_agreement is None:
             raise ValidationError("Terms of agreement cannot be null")
+        if input.customer_intent not in [
```

Comment:
Let's do:
`if input.customer_intent and input.customer_intent not in [...`

to enforce a default to the field so it does not break for current users.